packages/nextjs/src/config/types.ts

```diff
@@ -59,6 +59,14 @@ export type UserSentryOptions = {
 
   // Automatically instrument Next.js data fetching methods and Next.js API routes
   autoInstrumentServerFunctions?: boolean;
+
+  // Exclude certain serverside API routes or pages from being instrumented with Sentry. This option takes an array of
+  // strings or regular expressions.
+  //
+  // NOTE: Pages should be specified as routes (`/animals` or `/api/animals/[animalType]/habitat`), not filepaths
+  // (`pages/animals/index.js` or `.\src\pages\api\animals\[animalType]\habitat.tsx`), and strings must be be a full,
+  // exact match.
```

Comment:
m: Just something we should maybe think about: I am wondering if we should have users include `"pages/"` in the strings/regexes they provide here just to future-proof us against Next.js 13's `app/` folder. However, the more I think about it it shouldn't matter, as the routes in the `app/` folder will also just be routes like we have in the pages folder and I believe they can't collide - so we're probably good here.

Reply:
It's a good question, but I think you're right, a route is a route and you can't have two with the same route path. I also think that it's better that we abstract away the underlying file structure. That way, as users do move their routes from `pages/` to `app/`, they won't have to come and update this setting.